### ISSUE TITLE
[BUG]-ContentDocumentLink unexpected error in core:ExportFiles 

### ISSUE THREAD

**Issue by JimBTek** [forcedotcom/SFDX-Data-Move-Utility#531]
**Describe the bug**
trying to move around 300 mb across 900 files that are children to Account, and it works after reducing chunk size, but always fails to write the ContentDocumentLinks.  

**To Reproduce**
run the command with the core:ExportFiles

**Expected behavior**
ContentDocumentLinks should not fail

**export.json**
{    
    "objects": [
        {
            "operation": "Readonly",
            "externalId": "External_ID__c",
            "query": "SELECT External_ID__c FROM Account WHERE LastModifiedDate > 2022-01-01T00:00:00.000+0000 ORDER BY ParentId",

            "afterAddons" : [
                {
                    "module": "core:ExportFiles",
                    "args" : {
                        "operation" : "Insert"
                    }
                }
            ]
        }
    ],
    "sourceRecordsCache": "FileCache",
    "promptOnMissingParentObjects": false,
    "promptOnUpdateError": false,
    "promptOnIssuesInCSVFiles": false,
    "maxChunkSize": 10000000
}

**Log file**
{ContentVersion} The Insert API operation has been finished.
[core:ExportFiles] {Account} Processing chunk #19 of 48 items.
In progress... Completed 0/48 records.
In progress... Completed 10/48 records.
In progress... Completed 20/48 records.
In progress... Completed 30/48 records.
In progress... Completed 40/48 records.
In progress... Completed 48/48 records.
{ContentVersion} The Insert API operation has been started using REST API.
[Job# REST:Insert] {ContentVersion} The job has been created. Uploading data ...
{ContentVersion} The Insert API operation has been finished.
[core:ExportFiles] {Account} Total 968 records have been processed, 0 records failed.
[core:ExportFiles] {Account} Creating target ContentDocumentLink records ...
[core:ExportFiles] {Account} There are 968 records to process.
Command in progress ...... done
[ERROR] Execution of the command was ABORTED due to unexpected error: Cannot convert undefined or null to object.
Execution of the command sfdmu:run has been finished. Exit code 1 (COMMAND_UNEXPECTED_ERROR).

**_target.csv** file.
not relevant, org to org

**Comment by hknokh** [comment]
Hello.

I checked your issue but can't reproduce the issue.

I have not option to check the Add-On for extremely large volume of the files, but I tried it with 2 chunks of files and it passed the job successfully including creation of ContentDocumentLinks

Make sure you are using the latest version of the sfdmu and need to debug this issue on your side.

------------
I've paid attention that you put  globally the parameter "maxChunkSize"
It's NOT the global script parameter, but it is one of the ExportFiles add-on arguments like this:

"afterAddons" :  [
      {
	      "module": "core:ExportFiles",
	      "args" : {
		      "operation" : "Insert",
		      **"maxChunkSize": 10000000**
	      }
      }
]
